src/app/api/public/v1/notifications.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.api.dependencies.database import get_db
+from app.api.public.v1.schemas import PaymentNotificationSchema
+from app.integrations.yookassa import yookassa_client
+from app.models import ObjectDoesNotExistError, TransactionStatusEnum, Transaction
+
+router = APIRouter()
+logger = logging.getLogger("notification")
+
+
+@router.post("/yookassa/on-after-payment", description="Callback from yookassa")
+async def on_after_payment(
+    payment_data: PaymentNotificationSchema, db_session: AsyncSession = Depends(get_db)
+):
+    transaction_id = payment_data.object.id
+    transaction_data = yookassa_client.get_transaction(transaction_id)
```

Comment:
все сетевые запросы надо оборачивать в трай экзепт, т.к. могут быть сетевые ошибки, т.е. надо

```suggestion
try:
    transaction_data = await yookassa_client.get_transaction(transaction_id)
except YookassaHttpClientError:
    raise HTTPException(
            status_code=status.HTTP_424_FAILED_DEPENDENCY,
            detail="Yookassa unavailable.",
        )
```